src/libcmd/installables.cc

```diff
@@ -395,44 +395,56 @@ static StorePath getDeriver(
 struct InstallableStorePath : Installable
 {
     ref<Store> store;
-    StorePath storePath;
+    DerivedPath req;
```

Comment:
This is not actually a single path, despite the type name.
I'm ok to leave that as tech debt for now.
See https://github.com/NixOS/nix/issues/6507#issuecomment-1295822346

Reply:
It does actually need to be multiple paths now because of `^foo,bar` and `^*`.